src/apns_connection.erl

```diff
@@ -58,11 +60,15 @@
 -type host()         :: string() | inet:ip_address().
 -type path()         :: string().
 -type notification() :: binary().
--type type()         :: cert | token.
+-type type()         :: certdata | cert | token.
+-type keydata()      :: {'RSAPrivateKey' | 'DSAPrivateKey' | 'ECPrivateKey' | 'PrivateKeyInfo'
```

Comment:
Line 64 is too long: -type keydata()      :: {'RSAPrivateKey' | 'DSAPrivateKey' | 'ECPrivateKey' | 'PrivateKeyInfo'.